app/assets/javascripts/components/activity/activity_table.cjsx

```diff
@@ -39,9 +43,9 @@ ActivityTable = React.createClass(
 
     drawers = @state.activity.map (revision) ->
       courses = revision.courses.map (course) ->
-        <li><a href="/courses/#{course.slug}">{course.title}</a></li>
+        <li key={Math.random() * course.id}><a href="/courses/#{course.slug}">{course.title}</a></li>
 
-      <tr className='activity-table-drawer'>
+      <tr key={Math.random() * revision.key} className='activity-table-drawer'>
```

Comment:
Won't this generate new keys every time this component rerenders? If so, will changing the keys have performance implications (such as forcing React to recreate each of those elements instead of reusing the existing one)?

Maybe there's some other way to ensure unique keys without introducing random numbers?